java/ql/automodel/publish.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/sh
+set -e
+
+AUTOMODEL_ROOT="$(dirname $0)"
```

Comment:
I know that the original script has this same line (which I wrote a couple of years ago), but not sure if it's the best. This script is only running successfully from the `codeql/java/ql/automodel` directory. Running in the repo root is failing for some reason.

Reply:
Ah...I think the path in line 5 needs to be normalized. Running from the repo root will make `WORKSPACE_ROOT=java/ql/automodel/../../..`, but it really needs to be `WORKSPACE_ROOT=.`.

```suggestion
AUTOMODEL_ROOT="$(realpath "$(dirname $0)")"
```

Reply:
I have changed this to do:
```
AUTOMODEL_ROOT="readlink -f $(dirname $0)"
WORKSPACE_ROOT="$AUTOMODEL_ROOT/../../../.."
```

Happy to switch to `realpath` as well if that is better.

Reply:
Also updated the solorigate script with this change.